utility_analysis_new/combiners.py

```diff
@@ -517,7 +517,7 @@ class AggregateErrorMetricsAccumulator:
     """ Accumulator for AggregateErrorMetrics.
 
     All fields in this dataclass are sums across partitions"""
-    num_partitions: int
+    expected_kept_partitions: float
```

Comment:
nit: `kept_partitions_expected` for consistency with other fields here

Reply:
Good point. Done